mandatory/3-function-output.js

```diff
@@ -2,16 +2,24 @@
 function getRandomNumber() {
   return Math.random() * 10;
 }
-
+ //*
+// The random() method returns a random value that is greater than or equal to 0.0 and less than 1.0.
+//*
 // Add comments to explain what this function does. You're meant to use Google!
 function combine2Words(word1, word2) {
   return word1.concat(word2);
 }
+//*
+// This function will join the two words without space between them.
+//*
 
 function concatenate(firstWord, secondWord, thirdWord) {
   // Write the body of this function to concatenate three words together.
   // Look at the test case below to understand what this function is expected to return.
+  return firstWord.concat(" ", secondWord).concat(" ", thirdWord);
```

Comment:
I like the way you've called `.concat()` on the result of `.concat()`! Are you able to explain why this works?